test/integration/placement/placement_test.go

```diff
@@ -284,55 +287,113 @@ var _ = ginkgo.Describe("Placement", func() {
 			assertCreatingClusterSet(clusterSet1Name, "vendor", "openShift")
 			assertCreatingClusterSetBinding(clusterSet1Name, namespace)
 			clusters1 := assertCreatingClusters(clusterName+"1", 1, "vendor", "openShift")
-			assertCreatingPlacementWithDecision(placementName, namespace, noc(2), 1, clusterapiv1beta1.PrioritizerPolicy{}, []clusterapiv1beta1.Toleration{})
+			assertCreatingPlacementWithDecision(placementName, namespace, noc(2), 1, 1, clusterapiv1beta1.PrioritizerPolicy{}, []clusterapiv1beta1.Toleration{}, clusterapiv1beta1.GroupStrategy{})
 
 			// add more clusters
 			ginkgo.By("Add the cluster")
 			clusters2 := assertCreatingClusters(clusterName+"2", 1, "vendor", "openShift")
 
-			assertNumberOfDecisions(placementName, namespace, 2)
+			assertNumberOfDecisions(placementName, namespace, 2, 1)
 			assertPlacementConditionSatisfied(placementName, namespace, 2, true)
 
 			ginkgo.By("Delete the cluster")
 			assertDeletingClusters(clusters1...)
-			assertNumberOfDecisions(placementName, namespace, 1)
+			assertNumberOfDecisions(placementName, namespace, 1, 1)
+
+			assertDeletingClusters(clusters2...)
+			assertNumberOfDecisions(placementName, namespace, 0, 1)
+		})
+
+		ginkgo.It("Should update the decision group once clusters belong to labelselector clusterset are added/deleted", func() {
```

Comment:
I think we also have a static group field? should we also test it?

Reply:
Updated the case.